src/desktop/config.coffee

```diff
@@ -99,11 +99,14 @@ module.exports =
   POSITRON_URL: 'http://localhost:3005'
   PREDICTION_URL: 'https://live.artsy.net'
   REFLECTION_URL: 'http://artsy-reflection.s3-website-us-east-1.amazonaws.com/__reflection/forceartsynet'
+  BURST_REQUEST_LIMIT: 15,
+  BURST_REQUEST_EXPIRE: 1,
+  BURST_REQUEST_BLOCK_FOR: 90,
   REQUEST_EXPIRE: 60
   REQUEST_LIMIT: 300
   REQUEST_PER_INSTANCE_FALLBACK: 100
-  REQUEST_PER_INSTANCE_LIMIT: 301
-  REQUEST_PER_INSTANCE_EXPIRE: 60
+  REQUEST_PER_INSTANCE_LIMIT: 100
+  REQUEST_PER_INSTANCE_EXPIRE: 300
```

Comment:
Not a blocker, might be good to have an inline comment indicating the unit (seconds) here and on other lines that indicate a duration.

It's prob obvious right now, but if someone's adjusting these in the heat of battle, that could be helpful.